src/donationswap.py

```diff
@@ -598,6 +598,13 @@ def _get_match_score(self, offer_a, offer_b, db):
 
 		#xxx only count as "benefit" if own charity isn't tax-deductible, but other donor's one is
 		#    (otherwise we would reward pointless swaps, where both donors already get their tax back)
+		#    J -> we can start by checking whether we would have gotten a benefit by donating to our chosen charity anyway
+		a_would_have_benefit = entities.CharityInCountry.by_charity_and_country_id(offer_a.charity_id, offer_a.country_id) is not None
+		b_would_have_benefit = entities.CharityInCountry.by_charity_and_country_id(offer_b.charity_id, offer_b.country_id) is not None
```

Comment:
Shouldn't this be `would_have_benefited` (since it's used as a verb in the other variable down below)?

There's absolutely no need to change it... I'm just asking.